java/src/server/commands/game/ListAICommand.java

```diff
@@ -22,12 +24,17 @@
      */
     @Override
     public CommandExecutionResult execute() {
-        return null;
+        try {
+            return Config.facade.listAI(0);
+        } catch (ListAIException e) {
+            e.printStackTrace();
+            return new CommandExecutionResult("Error listing AI types!");
```

Comment:
If an error occurs, you need to call `.triggerError(int)` on the `CommandExecutionResult` object, passing in a status code that will be returned to the client. Otherwise the client will receive a 200.